spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -484,20 +488,40 @@ public void open() {
     intp.setContextClassLoader();
     intp.initializeSynchronous();
 
-    completor = new SparkJLineCompletion(intp);
+    numReferenceOfSparkContext.incrementAndGet();
```

Comment:
should this be under try {} catch {}? if someone fails between line 493-523, shouldn't this revert the ref count?